docs/output-targets/custom-elements-bundle.md

```diff
@@ -0,0 +1,200 @@
+---
```

Comment:
@alicewriteswrongs I think this got re-added on the rebase. Can you rm it?

Reply:
🤦‍♀️